lib/main/itoStreamController.ts

```diff
@@ -24,8 +26,12 @@ export class ItoStreamController {
   private isCancelled = false
   private configQueue: TranscribeStreamRequest[] = []
   private abortController: AbortController | null = null
+  private interactionId: string | null = null
 
-  public async initialize(mode: ItoMode): Promise<boolean> {
+  public async initialize(
+    mode: ItoMode,
+    interactionId: string,
```

Comment:
I'm not a huge fan of sprinkling interactionId throughout the code. Can we try to keep it isolated to InteractionManager?